src/components/AddressSearch/CurrentLocationButton.tsx

```diff
@@ -9,21 +8,9 @@ import useStyleUtils from '@hooks/useStyleUtils';
 import useThemeStyles from '@hooks/useThemeStyles';
 import getButtonState from '@libs/getButtonState';
 import colors from '@styles/theme/colors';
+import type {CurrentLocationButtonProps} from './types';
 
-const propTypes = {
-    /** Callback that runs when location button is clicked */
-    onPress: PropTypes.func,
-
-    /** Boolean to indicate if the button is clickable */
-    isDisabled: PropTypes.bool,
-};
-
-const defaultProps = {
-    isDisabled: false,
-    onPress: () => {},
-};
-
-function CurrentLocationButton({onPress, isDisabled}) {
+function CurrentLocationButton({onPress, isDisabled}: CurrentLocationButtonProps) {
```

Comment:
Missing default value:
```suggestion
function CurrentLocationButton({onPress, isDisabled = false}: CurrentLocationButtonProps) {
```